blocklytest/src/androidTest/java/com/google/blockly/model/FieldNumberTest.java

```diff
@@ -15,27 +15,38 @@
 
 package com.google.blockly.model;
 
-import com.google.blockly.android.MockitoAndroidTestCase;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
 
 /**
  * Tests for {@link FieldNumber}.
  */
-public class FieldNumberTest extends MockitoAndroidTestCase {
+public class FieldNumberTest {
     private static final String FIELD_NAME = "FIELD_NAME";
 
     private FieldNumber mField;
 
-    @Override
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Before
     public void setUp() throws Exception {
-        super.setUp();
         mField = new FieldNumber(FIELD_NAME);
     }
 
+    @Test
     public void testConstructor() {
         assertEquals(FIELD_NAME, mField.getName());
-        assertEquals(Field.TYPE_NUMBER, mField.getType());
+        assertEquals(Field.TYPE_NUMBER, mField.getType(), 0d);
```

Comment:
The assertEquals that was present was deprecated. The new one takes a precision as well, and was failing tests for me, so I've moved this class to use the new one as part of this change.